src/client/pfs.go

```diff
@@ -234,6 +235,25 @@ func (c APIClient) inspectCommit(repoName string, commitID string, blockState pf
 // `number` determines how many commits are returned.  If `number` is 0,
 // all commits that match the aforementioned criteria are returned.
 func (c APIClient) ListCommit(repoName string, to string, from string, number uint64) ([]*pfs.CommitInfo, error) {
```

Comment:
As a side note, it might be useful to have a commit range abstraction that holds the from, to, and number of commits.

Reply:
We do actually have a `CommitRange` proto message, but it's used in different places. It might make sense to group these together at some point.